ci-operator/jobs/open-cluster-management/observability-e2e-test/open-cluster-management-observability-e2e-test-release-2.3-postsubmits.yaml

```diff
@@ -74,6 +74,8 @@ postsubmits:
           value: "2.3"
         - name: OSCI_IMAGE_REMOTE_REPO_SRC
           value: registry.ci.openshift.org/open-cluster-management
+        - name: OSCI_PUBLISH_DELAY
```

Comment:
I do not think `grafana-dashboard-loader` needs 20 min delay to publish image. the `grafana-dashboard-loader` image can be built quickly.